2021/CifarelliDiFranco/dpa.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+
+import pyshark
+import utils
+import sys
+
+def main():
```

Comment:
A cosa serve questo file?

Reply:
Serve per effettuare una post analysis del traffico catturato live (flag -pa).

Reply:
Ok ma non vedo dove e' usato nel progetto, quindi lo toglierei. E darei un nome migliore ai files per capire qual'e' il main program.

Reply:
In che senso dove è usato? Se viene specificato il flag -pa viene chiamato il file dpa.py con il .pcap risultante dall'esecuzione..